### ISSUE TITLE
Deactivating this extension on Tex files

### ISSUE THREAD

**Issue by tholop** [YellPika/vscode-latex-input#4]
Apparently, the setting `latex-input.selector` defines on which files the extension should be activated. On the contrary, I would like to specify some files on which the extension should *not* be activated. Indeed, I'd like to replace the input everywhere, except `.tex` files.

Please, is it supported in the current settings? If yes, would you mind providing a code snippet showing how to update `settings.json`? Thanks!

**Comment by YellPika** [comment]
The `selector` must be an instance of a `DocumentFilter`. See [here](https://code.visualstudio.com/api/references/vscode-api#DocumentFilter) for details.

As far as I can tell, what you're asking for is not possible. The best you can do at the moment is manually list all the types of files you want the extension enabled on, e.g.

```
"latex-input.selector": [
  { "language": "plaintext" },
  { "language": "python" },
  { "language": "rust" }
]
```

**Comment by tholop** [comment]
Thank you!

**Comment by YellPika** [comment]
Fixed by 2024d486db33b3b144f4a9b947d5736d89068b32. The extension can now be enabled/disabled on a per-language basis, e.g.

```json
"[latex]": {
  "latex-input.enabled": false
}
```
